models/README.md

```diff
@@ -0,0 +1,7 @@
+# README
+
+This directory contains serializations of models in various formats, m.n. the LIonWeb JSON and Ecore XML formats:
```

Comment:
It seems to me that the directory contains JSON files but not XML files

Reply:
In a couple of PRs, there will also be XML files ;) I'll leave the doc as-is, otherwise I'll have to revert it twice.